packages/keystone/graphiql/graphiql.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Playground - __REPLACE_TO_YOUR_SERVER_URL__</title>
```

Comment:
```suggestion
    <title>Playground</title>
```

Reply:
Or even condo playground)

Reply:
The point here is to be able to differentiate between sandbox tabs of different applications.
This is exactly how it works for us now.
However, we can forget about this moment)